test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,38 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNotNull() throws Exception {
+        // empty list
+        assertHasNoNull();
+
+        // only one object
+        assertHasAnyNull((Object) null);
+        assertHasNoNull(1);
+        assertHasNoNull("");
+        assertHasNoNull("abc");
+
+        // all objects unique
+        assertHasNoNull("abc", "ab", "a");
+        assertHasNoNull(1, 2);
+        assertHasAnyNull("abc", "ab", "a", null);
+        assertHasAnyNull(1, 2, null);
+        assertHasAnyNull(null, "a");
+        assertHasAnyNull(null, null);
+        assertHasAnyNull(null, 1, "a", null);
+    }
+
+
+    public void assertHasAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
No need to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.

Reply:
so it is better to keep the assert inside each method?